extensions/interactions/DragAndDropSortInput/directives/drag_and_drop_sort_input_interaction_directive.html

```diff
@@ -59,29 +47,34 @@
     position: relative;
   }
 
+  .interaction-container {
```

Comment:
You can't have a class defined in one place and used in a different place.

Rename this accordingly and put the rule in the supplemental-card directive file.

Reply:
It is the generic class and the CSS attributes will affect other interactions as well. So, its attributes have been set for Drag and drop only in its Html template.

Reply:
Renamed the class name.

Reply:
Maybe you can use ng-class to set this class only when interactionId is drag and drop?? Then you can define it in this template.

Reply:
Special casing is a good idea. If that's the case, name the class something related to drag-and-drop, and do an ng-class as @prasanna08 suggests. 

Also, add a comment in the supplemental card template HTML explaining that more details about the class can be found in the file XXX. And add a comment on the CSS rule explaining really clearly what's going on, since this is very unusual.

Reply:
After lots of testing, I observed that auto-scrolling is behaving inconsistently after using `ng-class` (like: after refreshing sometimes it works and sometimes not) but without it, it's perfectly fine. So, a comment in the template may suffice our purpose here with using explicit class. What do you suggest?